<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1031</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>1001</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="start_page">
     <widget class="QPushButton" name="start_game">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>250</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="movie_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>1000</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit_game">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>400</y>
        <width>250</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="challenge">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>500</y>
        <width>250</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>挑战模式</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="main_page"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
